sdktests/common_tests_base.go

```diff
@@ -149,7 +151,19 @@ func (c commonTestsBase) withHTTPSTransportSkipVerifyPeer(t *ldtest.T) transport
 		})
 		return nil
 	})
-	return transportProtocol{"https", configurer}
+	return transportProtocol{"https", "https-skip-verify-peer", configurer}
+}
+
+func (c commonTestsBase) withHTTPSTransportVerifyPeerCustomCA(t *ldtest.T, customCAPath string) transportProtocol {
+	t.RequireCapability(servicedef.CapabilityTLSCustomCA)
+	configurer := helpers.ConfigOptionFunc[servicedef.SDKConfigParams](func(configOut *servicedef.SDKConfigParams) error {
+		configOut.TLS = o.Some(servicedef.SDKConfigTLSParams{
+			SkipVerifyPeer: false,
+			CustomCAFile:   customCAPath,
```

Comment:
file or path?  Pick one?

Reply:
It's a path.. but it points to a file.. I'm stuck in the depths of naming things.. (will use file)